src/templates/tags/tag-index.md

```diff
@@ -7,5 +7,5 @@ book_path: /web/{{section}}/_book.yaml
 # {{ title }} {: .page-title }
 
 {{#each tags}}
-* [{{this}}](/web/updates/tags/{{this}})
+* [{{this}}](/web/{{section}}/tags/{{this}})
```

Comment:
@petele actually, this doesn't work because it seems like `section` isn't defined inside the loop, even though it is defined at the top of the template. Any idea what syntax to use here?